README.md

```diff
@@ -1,60 +1,69 @@
-# azure-collector
+# Alert Logic Microsoft Office 365 Log Collector
 
 [![Build Status](https://secure.travis-ci.org/alertlogic/azure-collector.png?branch=master)](http://travis-ci.org/alertlogic/azure-collector)
 
 AlertLogic Office 365 Log Collector
 
 # Overview
 
-This repository contains Azure Web application Node.js source code and an ARM template for setting up a data collector in Azure which will collect and forward Office 365 log data to the Alert Logic Cloud Defender Log Manager (LM) feature.
+This repository contains the Microsoft Azure web application Node.js source code and an Azure Resource Manager (ARM) template to set up a data collector in Azure, which collects and forwards Microsoft Office 365 log data to Alert Logic Log Management. 
 
 # Installation
 
-Installation requires the following steps:
+To perform the set up required to grant Alert Logic permission access to collect Office 365 logs, you must have access to the following:
+
+* A Microsoft Office 365 subscription with administrative privileges
+* A Microsoft Azure account with administrative privileges
+* An Alert Logic user account with administrative privileges
+<!--The information above appears in the AL Docs. If this is still a prerequisite, we need to call it out here. So I did. -->
 
-1. Register a new O365 web application in O365 portal for collecting O365 logs.
-1. Set up the required Active Directory security permissions for the application to authorize it to read threat intelligence data and activity reports for your organization.
-1. Create an Access Key that will allow the application to connect to the Alert Logic Cloud Defender and Cloud Insight backend.
-1. Download and deploy a custom ARM template to Microsoft Azure to create functions for collecting and managing O365 log data
-1. Verify the installation was successful using the Alert Logic UI under Configuration -> Deployments -> All Deployments -> Log Sources -> Filter by `Push (Office 365, CloudWatch)` collection method.
 
 ## Register a New O365 Web Application in O365
 
-In order to install O365 Log collector:
+In the Office 365 portal, you must register a new Office 365 web application to collect Office 365 logs. 
+
+**To register an Office 365 web application to collect logs:**
 
-1. Log into [O365 portal](https://portal.office.com) as AD tenant administrator.
+1. Log into the [Office 365 portal](https://portal.office.com) as an Active Directory tenant administrator.
 1. Navigate to `Admin Centers` and `Azure AD`.
-1. On the left side panel click `Azure Active Directory` then select `App Registrations`.
-1. Click `+ New application registration`, fill in configuration parameters and click `Create`:
-    * `Name` - for example `alo365collector`.
+<!-- I need to clarify step 2. Where are these options? -->
```

Comment:
[Here](https://support.appriver.com/AvatarHandler.ashx?radfile=%5CUsers%5C229%5CO365%5Cazure001%2Epng) is the example.

Reply:
Thanks!